src/utils/eth_transaction.cairo

```diff
@@ -258,4 +272,65 @@ namespace EthTransaction {
         finalize_keccak(keccak_ptr_start, keccak_ptr);
         return ();
     }
+
+    // @notice Recursively parses the RLP-decoded access list.
+    // @param parsed_list The pointer to the next free cell in the parsed access list.
+    // @param list_len The remaining length of the RLP-decoded access list to parse.
+    // @param list_items The pointer to the current RLP-decoded access list item to parse.
+    // @return The number of parsed access list entries.
+    func parse_access_list{range_check_ptr}(
+        parsed_list: model.AccessListItem*, list_len: felt, list_items: RLP.Item*
+    ) -> felt {
+        alloc_locals;
+        if (list_len == 0) {
+            return 0;
+        }
+
+        // Address
+        let address_item = cast(list_items.data, RLP.Item*);
+        let address_ptr = address_item.data;
+        let address_len = address_item.data_len;
+        let address = Helpers.bytes20_to_felt(address_ptr);
```

Comment:
can just do `Helpers.bytes20_to_felt(address_item.data)`, the two `address_X` are useless